.github/workflows/src/sdk-breaking-change-labels.js

```diff
@@ -0,0 +1,186 @@
+// @ts-check
+import { sdkLabels } from "../../src/sdk-types.js";
+import { LabelAction } from "../../src/label.js";
+
+const wfName = 'sdk-breaking-change-labels';
+
+/**
+ * @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments
+ * @returns {Promise<{labelName: string, labelAction: LabelAction, issueNumber: number, state: string}>}
+ */
+export async function getLabelAndAction({ github, context, core }) {
+  let issue_number = NaN;
+  let check_run_id = process.env.CHECK_RUN_ID || context.payload.check_run?.id;  
+  let labelAction;
+  let labelName = "";
+
```

Comment:
Please use the pattern defined here for extracting inputs:

https://github.com/Azure/azure-rest-api-specs/blob/main/.github/workflows/src/arm-auto-signoff-preview.js

Top-level function called by the workflow YAML, that extracts inputs and calls a helper method to do the actual work.  This makes our workflows more consistent, and allows you to test the business logic separately from the logic to extract the inputs.

Thus, I think a lot of the code below should be moved into context.js.  Any workflow that needs to trigger on a DevOps pipeline and extract inputs from it will need similar code, right?

Reply:
Made some input properties as optional in context.js and added common logic to there for check_run case.